File: utbot-framework/src/main/java/org/utbot/quickcheck/generator/GeneratorConfigurationException.java

package org.utbot.quickcheck.generator;

/**
 * Raised if a problem arises when attempting to configure a generator with
 * annotations from a property parameter.
 *
 * @see Generator#configure(java.lang.reflect.AnnotatedType)
 */
public class GeneratorConfigurationException extends RuntimeException {
    private static final long serialVersionUID = 1L;

    public GeneratorConfigurationException(String message) {
        super(message);
    }

    public GeneratorConfigurationException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: utbot-framework/src/main/kotlin/org/utbot/quickcheck/generator/GeneratorConfigurationException.kt
package org.utbot.quickcheck.generator

/**
 * Raised if a problem arises when attempting to configure a generator with
 * annotations from a property parameter.
 *
 * @see Generator.configure
 */
class GeneratorConfigurationException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID = 1L
    }
}